app/components/finance/statements/clawbacks.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module Finance
+  module Statements
+    class Clawbacks < BaseComponent
```

Comment:
I wonder if we should separate the clawbacks logic for ects/mentor and make the decision of which ones to show outside this component? maybe have a base and then separate the different types?

Reply:
Refactored this, now the decision made in the calculators